wafer/talks/models.py

```diff
@@ -24,6 +25,24 @@ def render_author(author):
     return '%s (%s)' % (author.userprofile.display_name(), author)
 
 
+def authors_help():
+    _ = ugettext  # This function will be wrapped for lazy evaluation
+    text = []
+    text.append(_("The speakers presenting the talk."))
+    if not settings.WAFER_PUBLIC_ATTENDEE_LIST:
+        text.append(_(
+            "To ensure attendee privacy, you will only be able to see "
+            "yourself and authors that have been added to the talk by the "
+            "conference organisers. "
+            "If you will have other co-authors, add a note in the notes "
+            "field, so the organisers can add them to your talk."
+        ))
+    text.append(_(
+        "You, as the talk submitter, will be the talk's corresponding author."
```

Comment:
With the full text displayed, this note gets a bit lost at the end of it. I think it should be emphasised in some way - perhaps wrapped in a <strong> tag or similar?